test/parallel/test-net-server-listen-options.js

```diff
@@ -36,46 +24,51 @@ const listenOnPort = [
 ];
 
 {
-  const portError = /^RangeError: "port" argument must be >= 0 and < 65536$/;
+  const assertPort = () => {
+    return common.expectsError({
+      code: 'ERR_SOCKET_BAD_PORT',
+      type: Error
```

Comment:
This was a `RangeError` before?

Reply:
Good point, this one should be a `RangeError`

Reply:
my bad. I corrected this, now the error types in lib/net.js have the correct value.